lib/puppet/provider/openldap_database/olc.rb

```diff
@@ -236,7 +248,7 @@ def create
     t << "olcAccess: to *\n"
     t << "  by self write\n"
     t << "  by dn=\"cn=admin,#{resource[:suffix]}\" write\n"
-    t << "  by * read\n"
+    t << "  by * read\n"    
```

Comment:
Yes, you're right. but I could not see these line from my source.
It's strange.